frontend/packages/dev-console/src/components/monitoring/metrics/MetricsDropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as fuzzy from 'fuzzysearch';
+import { RootState } from '@console/internal/redux';
+import { Dropdown } from '@console/internal/components/utils';
+import { queryBrowserRunQueries, queryBrowserPatchQuery } from '@console/internal/actions/ui';
+import { getActiveNamespace } from '@console/internal/reducers/ui';
+import { QueryObj } from '@console/internal/components/monitoring/query-browser';
+import { metricsQuery, getTopMetricsQueries } from '../queries';
+import './MetricsDropdown.scss';
+
+const ADD_NEW_QUERY = '#ADD_NEW_QUERY#';
+
+interface StateProps {
+  namespace: string;
+}
+
+export type MetricsDropdownProps = {
+  patchQuery: (patch: QueryObj) => void;
+  runQueries: () => never;
+  namespace: string;
+};
+
+const MetricsDropdown: React.FC<MetricsDropdownProps> = ({ patchQuery, runQueries, namespace }) => {
+  const items = metricsQuery;
+  const title: React.ReactNode = 'Select Query';
```

Comment:
if we are not accepting `title` to be changed from parent component, then the type can be `string`